recipes/nanorange/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class NanorangeConan(ConanFile):
+    name = "nanorange"
+    version = "20191001"
```

Comment:
```suggestion
```
don't specify a version in the recipe when using the generic folder `all`

Reply:
Ok, I clearly need to reread the documentation, because that is why I chose that option there should most definitely be a version.

Reply:
no problem, actually we could added a hook to check it automatically :wink:

Reply:
Ah, now I get it.
Since it is already specified in the `config.yml` file it shouldn't be included in the `conanfile.py`